### PR TITLE
Fixed Behat tests for Turnitin Integrity

### PR DESCRIPTION
**Pull request by pdubey2019**

### CODE REVIEW

File: assignment.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@plugin @plagiarism @plagiarism_turnitinsim @plagiarism_turnitinsim_assignment
+@plugin @plagiarism @plagiarism_turnitinsim @plagiarism_turnitinsim_assignment @turnitinsim_smoke
```
Comment (jmcgettrick) [inline assignment.feature]: I would call that `plagiarism_turnitinsim_smoke` for consistency

File: assignment.feature [hunk 2]
```diff
@@ -39,7 +39,7 @@ Feature: Plagiarism plugin works with a Moodle Assignment
     And I follow "Test assignment name"
     Then I should see "Grading summary"
 
-  @javascript
+  @javascript @_file_upload
```
Comment (jmcgettrick) [inline assignment.feature]: remove the first underscore

File: assignment_drafts.feature [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@plugin @plagiarism @plagiarism_turnitinsim @plagiarism_turnitinsim_assignment @plagiarism_turnitinsim_assignment_drafts
+@plugin @plagiarism  @plagiarism_turnitinsim_assignment @plagiarism_turnitinsim_assignment_drafts
```
Comment (jmcgettrick) [inline assignment_drafts.feature]: Do we not want this as part of the whole suite?

File: assignment_hideidentities.feature [hunk 4]
```diff
@@ -38,7 +38,7 @@ Feature: Plagiarism plugin works with a Moodle Assignment
     And I follow "Test assignment name"
     Then I should see "Grading summary"
 
-  @javascript
+  @javascript @_file_upload @runme
```
Comment (jmcgettrick) [inline assignment_hideidentities.feature]: runme ?

File: assignment.feature (hunk 2 above)
Comment (dwinn) [reply assignment.feature]: The underscore is correct. No idea why but it is.

File: forum.feature [hunk 5]
```diff
@@ -34,22 +34,21 @@ Feature: Plagiarism plugin works with a Moodle forum
...
   Scenario: Add a post to a discussion with a file attached and retrieve the originality score
     Given I log out
     # Student creates a forum discussion and replies to original post.
     And I log in as "student1"
     And I am on "Course 1" course homepage
     And I follow "Test forum"
-    And I click on "Add a new discussion topic" "button"
-    And I click on "I accept the Turnitin EULA" "button"
+    And I follow "Add a new discussion topic"
+    #And I click on "I accept the Turnitin EULA" "button"
```
Comment (dwinn) [inline forum.feature]: I don't think this is meant to be commented out.

File: assignment_drafts.feature (hunk 3 above)
Comment (dwinn) [reply assignment_drafts.feature]: I agree, all tests should have the @plagiarism_turnitinsim tag so we can run the full suite.

File: forum.feature (hunk 5 above)
Comment (pdubey2019) [reply forum.feature]: I think by mistake I have left it there. I have double checked now there is no eula present at this page

Comment (dwinn) [reply forum.feature]: Odd that there's no EULA there. I thought the step would only accept it if the EULA is shown but I checked it and don't see logic for that. Does the test pass if you run it on its own?

Comment (pdubey2019) [reply forum.feature]: Yeah it worked fine for me. I ran it with keeping this step and it failed there but when I removed the step it just worked fine.

**Pull request merged**